packages/orm/entity/Portfolio.ts

```diff
@@ -20,6 +20,19 @@ export enum DodComponent {
   OSD_PSAS = "OSD_PSAS",
   NSA = "NSA",
 }
+export interface IPortfolio {
+  name: string;
+  description: string;
```

Comment:
Should `description` be optional? (same for the one below).

Reply:
I am going to fix this, but I am expecting Jason to overwrite this.

Reply:
Oh okay. Yeah, I was thinking this might be overwritten, since Jason is working on it.